lib/BigBlueButton/API.php

```diff
@@ -188,6 +188,10 @@ private function buildMeetingParams(Room $room, Presentation $presentation = nul
 		$invitationUrl = $this->urlHelper->linkToInvitationAbsolute($room);
 		$createMeetingParams->setModeratorOnlyMessage($this->l10n->t('To invite someone to the meeting, send them this link: %s', [$invitationUrl]));
 
+		if (!empty($room->logoutURL)) {
```

Comment:
Should probably check if the logoutURL is empty. Anyway I would prefer to have the assignment at one place. Maybe

```
$createMeetingParams->setLogoutURL($room->logoutURL || $this->urlGenerator->getBaseUrl());
```